src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,48 @@
+@import "./utils/_variables" ;
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  width: 706px;
```

Comment:
This is like a magic number, it is not clear why is it set to 706px and if you change for instance dimensions of calendar-day things will break up.
You should calculate the width basing on your variables ;)